### PR TITLE
Send mail rent

### PR DESCRIPTION
**Pull request by gussiciliano**
### Sumary
- send mail when create a rent

### Trello Card
https://trello.com/c/yWyJUMAS/11-mandar-email-al-usuario-cuando-se-crea-un-rent

### CODE REVIEW

File: app/mailers/rent_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class RentMailer < ApplicationMailer
+
+  def new_record_notification(rent)
+    @rent = rent
+  	mail to: rent.user.email, subject: "New rent"
```
Comment (Dinuuu) [inline app/mailers/rent_mailer.rb]: Extra indented

File: app/serializers/rent_serializer.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class RentSerializer < ActiveModel::Serializer
+  attributes :id, :book_id, :user_id, :from, :to
+end
```
Comment (Dinuuu) [inline app/serializers/rent_serializer.rb]: new line missing

File: app/serializers/rent_serializer.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+class RentSerializer < ActiveModel::Serializer
+  attributes :id, :book_id, :user_id, :from, :to
```
Comment (Dinuuu) [inline app/serializers/rent_serializer.rb]: I think you need to serialize the whole user and the whole book

File: spec/fixtures/rent_mailer/new_record_notification [hunk 4]
```diff
@@ -0,0 +1,3 @@
+RentMailer#new_record_notification
```
Comment (Dinuuu) [inline spec/fixtures/rent_mailer/new_record_notification]: I think you don't need this file

File: spec/mailers/previews/rent_mailer_preview.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Preview all emails at http://localhost:3000/rails/mailers/rent_mailer
+class RentMailerPreview < ActionMailer::Preview
+
```
Comment (Dinuuu) [inline spec/mailers/previews/rent_mailer_preview.rb]: neither this one

File: spec/mailers/rent_mailer_spec.rb [hunk 6]
```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe RentMailerMailer, type: :mailer do
```
Comment (Dinuuu) [inline spec/mailers/rent_mailer_spec.rb]: Here you are just testing the mock mail

File: app/controllers/api/v1/rents_controller.rb [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+        render json: @rent
+      end
+
+      def show
+        @rent = Rent.find(params[:id])
+        render json: @rent
+      end
+
+      def create
+        @rent = Rent.new(user_id: params[:user_id], book_id: params[:book_id], from: params[:from], to: params[:to])
```
Comment (Dinuuu) [inline app/controllers/api/v1/rents_controller.rb]: Here make the changes we already talk in person

File: app/mailers/rent_mailer.rb (hunk 1 above)
Comment (gussiciliano) [inline app/mailers/rent_mailer.rb]: Oks

File: app/serializers/rent_serializer.rb (hunk 2 above)
Comment (gussiciliano) [inline app/serializers/rent_serializer.rb]: Oks

File: spec/fixtures/rent_mailer/new_record_notification (hunk 4 above)
Comment (gussiciliano) [inline spec/fixtures/rent_mailer/new_record_notification]: Oks

File: spec/mailers/previews/rent_mailer_preview.rb (hunk 5 above)
Comment (gussiciliano) [inline spec/mailers/previews/rent_mailer_preview.rb]: Oks

File: spec/mailers/rent_mailer_spec.rb (hunk 6 above)
Comment (gussiciliano) [inline spec/mailers/rent_mailer_spec.rb]: Oks

File: app/controllers/api/v1/rents_controller.rb (hunk 7 above)
Comment (gussiciliano) [inline app/controllers/api/v1/rents_controller.rb]: Oks

File: app/serializers/rent_serializer.rb (hunk 3 above)
Comment (gussiciliano) [inline app/serializers/rent_serializer.rb]: Oks

**Pull request merged**